### DISCUSSION TITLE
有辦法用6個檔案漢化但改字型嗎

### DISCUSSION THREAD

**Discussion by Au-Neptune** [GpointChen/FFXIVChnTextPatch-GP#104] [Q&A] [answered]
如題，想用別人漢化的6個檔案漢化但他的是4k字型，想問能漢化但字型是一般的嗎?
4k的看起來有點不舒服

**Comment by qwe3410475** [comment]
你下載不是4k的就好了啊

**Comment by Au-Neptune** [comment]
> 你下載不是4k的就好了啊

如果他有給的話我就不會來問了= =

**Comment by qwe3410475** [comment, marked as answer]
喔，我以為你是在說這個漢化
沒看到你說用別人的
000000.win32 的檔案是字體 拿掉裝這裡的也行

**Comment by GpointChen** [comment]
如上回覆，感謝。

關閉本串，如又有問題請開新串。
